cmd/mt-index-cat/out/tpl_pattern_custom.go

```diff
@@ -0,0 +1,132 @@
+package out
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+)
+
+type bucket struct {
+	chance int
+	fn     func(in string) string
+}
+
+// percentage chance, and function
+func patternCustom(in ...interface{}) string {
+	usage := func() {
+		fmt.Println("usage of patternCustom:")
+		fmt.Println("input | patternCustom <chance> <operation>")
```

Comment:
it needs to be made clear that `<chance> <operation>` can be repeated.  convention for that is
```
input | patternCustom <chance> <operation>[ <chance> <operation>...]
```